Ziyu Beta Reading (#3)
mienaikoe/ziyu: Chapters/3-1-Chapter15.md

## Chapter 15

<!--
- Radio, Voices
  - Melna confused
  - Passes listener to Melna
  - Melna hears static
    - Lin upset
    - Listeners only work on Ulukoma
      - "Oh shit. What do they do now?" moment

- Get sleep.
-->

"Melna, I think you should see something."

> Melna stopped her work abruptly and left the incandescent lamp breathing light onto her makeshift desk to approach a man sitting in the dark. His eyes were fixed on a smoothly polished plate before him while his hands gently fumbled with the damper controls of a large telescopic projector pointed toward the horizon. She appeared behind him, unseen but heard, and he made room for her without looking away.

Reviewer: Too much going on in this description. I had to read it 4 times. Break up your sentences maybe?

```suggestion
Melna stopped her work abruptly, leaving the incandescent lamp breathing light onto her makeshift desk. She approaches a man sitting in the dark; his eyes were fixed on a smoothly polished plate before him. While his hands gently fumbled with the controls of a large telescopic projector pointed toward the horizon. He made room for her without looking away when she appeared behind him, unseen but heard.
```

"Okay Jathi, lead the way," Melna asked.

"Look for a line of small dots rising up from the horizon."

Melna stared into the image plate, searching for any sign of a light in the darkness.

"It's just the sky," she said.

"Close your eyes for a moment, and then open them slowly."

She did as the observer instructed, closing her eyes for longer than she had the patience for, then opened them again. A tiny string of five dots as bright as Ara punctuated the thin yellow-orange of the horizon sky. Jathi swapped out lenses to the highest magnification he had and adjusted the dials ever so delicately to find the dots again. They were still faint, being so close to the horizon, but they had some speed to them and they were rising quickly.

"What are they?" she asked with new urgency.

"Spacecraft. It looks like Holrag has launched something behind Liffa."

> Flushed with anger, she rose from the plate and closed her eyes, facing the darkest corner of the room.

Reviewer: holding back tears?

"How large are they? Where are they going?" She asked.

The observer answered her, "I can't quite tell their size at this distance, but judging by the speed of the vehicles, I think they're on a similar trajectory as Liffa. I'll have more concrete numbers for you tomorrow."

Melna remained silent.

"What should we do, žah-Melna?,"

Melna took a few deep breaths to regain her composure.

"Holragi wouldn't need any more missions than the one Liffa is on unless something has gone terribly wrong."

"But we've been getting recordings of Liffa's voice every tova'la," said Jathi.

"If they aren't missions, then they must be weapons of some kind. They aren't waiting for Liffa to return."

> "Jathi," she turned and approached him. "I want you to record every dot, every time, and find a way to determine their size and trajectory. Pomri will be here tomorrow morning to get your updates before you switch shifts. If anything changes after that, wire your findings to me on the private senate channel. I'm calling for a debate in the evening."

Reviewer: > I'm calling for a debate in the evening.

srsly? they are going to debate about this? I don't know if the fate of the planet should depend on whoever is the best rhetoric. I know that's how it works here and now... but it's just.... *sigh*

"We shall keep a constant eye."

"And Jathi," she recalled, "Keep your lines open. Communication is the only way we'll get through this."

"You have my word, žah."

Jathi scurried away to fetch some empty film plates while Pomri rose from her desk and approached Melna.

"If I could interject. The senate still believes that Liffa's mission is the only way out of this mess. They won't listen to an appeal."

"This goes beyond my family. This is about Holrag's secrets. All of Palinan is relying on them to lead the way, and still they keep us in the dark."

"Holrag has a plan. We must play by their rules."

"It's far too much power for one province to wield."

"Perhaps you're right. But our only other option is war."

Melna sighed, exasperated at how the world had come to this. "We need to find a way to force Holrag's hand."

> "Painting them as a villain will be met with hostility. There is very little to gain with a diplomatic approach."

Reviewer: starting to lose track of who is saying what here again

"What do you suggest then? We cower in fear, waiting for the day Holrag saves us all?"

"You bowed to them when you sacrificed our grain. You bowed to them when you sent your own bundsmaman into their experiment. What did it gain for us?"

Melna looked at her with smoldering eyes and explained herself, "Our position in the senate has been maintained through the worst of times. The nations continue to share knowledge and wealth to the benefit of everyone."

"The senate? You don't find it strange that the conduit has been quiet lately?"

Melna sighed, letting her internal frustration simmer and settle.

"I have to go home."

Pomri threw her hands up in disappointment as Melna reached for a thick felted black shawl and opened the door of the observatory to leave. A chilly breeze clambered through the door and into Pomri's robes, forcing her to clasp her arms.

> Melna turned back to her and said, "Meet me tomorrow at the train station. Bring observer Jathi's findings."

Reviewer: > Bring observer Jathi's findings

???

"But the trains aren't running to Heiko."

"Bring your saffa. We'll ride to Rudjan and catch the second train there."

"That's ridiculous!"

Melna closed the door behind her, ignoring Pomri's complaints.

> A small anomin lantern hung from the outside wall where she'd left it in the morning. She picked it up to light it with a firestarter, and tread carefully down the muddy steps, avoiding the newly formed dew puddles. Her mind was littered with worries, all of them feeding her imagination of watching Liffa sobbing, fearful, and alone in a cramped machine far away from home. The wind threw a gust at her, and she wondered if it was cold in space. She imagined wrapping her arms around Liffa to keep hae warm, and it made her feel warmer too.

Reviewer: > avoiding the newly formed dew puddles.

plants are so large they form dew puddles?

"What is your name?" she asked gently.

The kaln could only muster a whisper through hae raspy voice, so Melna bent down closer to hear.

"Lin... Yu."

---

Melna filled the kitchen mantle with kindling from the garden and pressed her firestarter against it to start a fire. Linyu sat staring at the wall in the dimly lit great room. Thi gasped when thi smelled smoke, remembering the heat of the fire and the screams of panic, but quickly snapped out of it for fear of making hae host suspicious. Melna sensed something off about the kaln and moved to a large dew column at the center of the house to draw two cups of water from the spigot.

"You must be thirsty, Linyu. Have some water."

At first Linyu hesitated, staying still with the cup next to hae eyes, Melna holding them with obvious anxiety. When thi began to reach softly all around for where the cup might be, thi took it and with a small sip, winced as thi began to swallow. Melna immediately noticed, and rose to take the cup down and examine both hae neck and eyes closer.

"You have a lot of swelling here. What happened?"

"F-Fire," Linyu whispered.

> "Oh," she said, surprised to hear a lie. "You should know that I'm a great listener. My job is to listen to people talk all day. If you ever want to tell me what really happened, you can say so."

Reviewer: This feels weird to say.

Linyu paused for a moment to interpret the strangeness of what Melna had just said, and then continued setting down the trinket on the ground near hae and then feeling hae way back to the washroom. Hesitantly, Melna moved her gaze to the trinket. It was wooden with metal rod sticking out and intricate carvings on the top that seemed to have survived through a great number of trouble. She turned it over to find three jagged holes drilled into the bottom.

_What is this thing?_

She faintly heard some noise coming from it, and hesitantly put it up to her ear. The sound was static at first, but she continued to listen closer as the observers taught her to do with the projectors at the observatory, and through the static she began to make out the tones and rhythms of a voice.

Linyu startled her from a trance-like fixation, standing in front of her with Liffa's robes and hae own wet clothes in hae hands. Thi knew where the listener had gone, and carried a look of consternation. Thi fervidly felt for Melna's hands amidst her confusion and pulled the listener from them, taking no pause to hide it away in hae damp, dirty clothes.

She responded, "I'm sorry, I should have asked first."

> Linyu whispered to haeself, "Mom."

Reviewer: In this world, what's a mom?

"Linyu, may I come in?"

There was no response.

"Knock once if you want me to leave. Twice if you'd rather I stay here."

After a long stillness there came a shuffling and then a knocking against the door. _Tap...Tap._

Slightly relieved, she responded, "Okay. I'll stay then."

She decided to be deliberate. Another prying question might push Linyu to leave, and it had become far too cold for anyone to be outside for the night.

"I promise that I won't ask any more questions. And I won't steal your device. I just have some water for you here by the door."

Thi slowly creaked the door open just enough to listen for Melna and reach for the glass which Melna gave eagerly. Thi kept the door cracked like this, sipping on water and listening vigilantly for any changes or movements. A crack of light shone through and Melna could see the texture of Liffa's robes clothing Linyu's shoulders.

> "You know, my jundmaman..." she began, fighting off her own tears, "Thi used to run to the washroom whenever thi was upset too. I never did a great job comforting hae. The other one was always better at it."

Reviewer: > Thi used to run to the washroom whenever thi was upset too.

ah i see....

> The other one was always better at it.

Feels unnatural. "Kipnu was always better at it," sounds more realistic even if Melna is aware Linyu might not know who he is. People often speak with less awareness of their audience. Does Melna ever let her guard down?
